backend/inspirehep/indexer/cli.py

```diff
@@ -17,13 +17,22 @@
 from invenio_db import db
 from invenio_pidstore.models import PersistentIdentifier, PIDStatus
 from invenio_search import current_search
+from invenio_search.cli import index
 
 from inspirehep.indexer.tasks import batch_index
 from inspirehep.records.api import InspireRecord
 
 LOGGER = structlog.getLogger()
 
 
+@index.command("reindex")
+def reindex_overwrite(*args, **kwargs):
+    """WARNING: Invenio reindex command is outdated. Use `indexer reindex` command instead."""
+    click.echo(
+        "*INACTIVE* Invenio reindex command is outdated. Use `indexer reindex` command instead. *INACTIVE*"
+    )
```

Comment:
what's the point of doing this, why don't we simply define this to be our own `reindex`, instead of having a different group for that?

Reply:
This task is for blocking invenio reindex command. Our own reindex in in indexer reindex. I think that it's done like that to easily distinguish our indexer commands from invenio ones but I wasn't one who separated them and I don't recall when it happened. Please discuss things like that on Jira tasks, not during reviews, as it's a little too late already ;)

Reply:
It was done like this with a separate command as a workaround for the invenio indexer. I agree that it should have been raised during the refinement, but I didn't take part in it.

Reply:
> As it was mentioned in refinement already, when we are overwriting index group in inspirehep.cli it's already AFTER app is initialized so loading order doesn't matter here anymore.

Actually, this is exactly how it was before, we're overriding the ``reindex`` command but it didn't work https://github.com/inspirehep/inspirehep/blob/1defd3f6849dba6ae43253af2cb1bd64c7ac98ce/backend/inspirehep/indexer/cli.py#L71 It was the actual reason why we moved the commands in a sepate group (bad solution),  the  ``reindex`` command was pointing to ``invenio_inexer.cli`` instead of ``inspirehep.indexer.cli`` due to wrong entrypoints loading.

We have to solve the order issue and just add/override the ``invenio_indexer.cli``,  to have a unified cli. As we have discussed during the refinement the main problem is the entrypoint loading order. We have to just fix the order, add the ``remap`` command and override the ``reindex`` command , which was acutally a very valid point @pazembrz raised during the refiment. 

Also even if it's in the refinement, if we find during the reivew that something is not optimal or wrong we don't have to merge it. We can simply adapt or close the PR.

Reply:
But we never put this overwritten index group in inspirehep.cli, this is why it was depending form load order.

Reply:
I'm not sure I understand, maybe I'm missing something. The code seems exactly the same as before https://github.com/inspirehep/inspirehep/blob/1defd3f6849dba6ae43253af2cb1bd64c7ac98ce/backend/inspirehep/indexer/cli.py#L71

Reply:
please check inspirehep/cli.py